test/performance/odp_sched_perf.c

```diff
@@ -275,6 +297,17 @@ static int parse_options(int argc, char *argv[], test_options_t *test_options)
 		case 'j':
 			test_options->num_join = atoi(optarg);
 			break;
+		case 'C':
+			if (cache_stash_size >= MAX_CACHE_STASH)
+				continue;
+			if (sscanf(optarg, "%u,%u,%u,%u",
+				   &test_options->cache_stash[cache_stash_size].offset,
+				   &test_options->cache_stash[cache_stash_size].len,
+				   &test_options->cache_stash[cache_stash_size].region,
+				   &test_options->cache_stash[cache_stash_size].level) != 4)
+				ret = -1;
+			cache_stash_size++;
+			break;
```

Comment:
The configuration of stashing could get a bit more straightforward if more stuff was done at this parsing phase. The current string scanning (done here), value validation (done below) and configuration setting (done in `init_stash_params()`) could be fused to a function, e.g. `parse_cache_stash_config()`, which would fill directly to a `odp_cache_stash_config_t` at `test_options_t` level (multiple `-C` to same region/level would just override the previous), the new function then could take `optarg` and this new field as parameters. Basically what you've already done but fused together. The current `init_stash_params()` (to be renamed) would then just configure the scheduler.